packages/genesys-spark-components/src/components/stable/gux-button/gux-button.tsx

```diff
@@ -92,6 +106,12 @@ export class GuxButton {
       if (child.tagName === 'GUX-ICON') {
         return true;
       }
+    } else if (
+      children.length === 2 &&
+      children[0].tagName === 'GUX-ICON' &&
+      children[1].tagName === 'GUX-TOOLTIP'
+    ) {
+      return true;
```

Comment:
I found out that the gux-tooltip idea did cause an issue with icon-only buttons, making them wider because they didn't have the gux-icon-only class anymore. The gux-tooltip example page uses a text button so it wasn't seen there.
This logic fixes that.

Reply:
Might want to expand this to gux-tooltip-beta also

Reply:
Done.
I also noticed a couple other issues that I've fixed.
1. The padding was weird on the tooltip
2. The example ran into the same accessibility issues I noticed before. Updated the example in a way that fixes it.
3. Added examples of buttons with text using a descriptive tooltip too.

Reply:
For the icon buttons, the screenreader is still reading out the 'Visual of screen reader text' for me.

Reply:
`  <gux-button onclick="notify(event)">
    <gux-icon
      icon-name="fa/diamond-regular"
      screenreader-text="This will be read by a screen reader"
    ></gux-icon>
    <gux-tooltip-beta visual-only="true" aria-hidden="true"
      ><span slot="content"
        >Visual of screen reader text</span
      ></gux-tooltip-beta
    >
  </gux-button>`

I can get the screenreader to ignore the text if I add aria-hidden here but not if I add it to the host element in gux-tooltip. I don't know why that happens

Reply:
That is really strange since the whole point of the `visualOnly` is that it adds `aria-hidden` to the tooltip.. is this another case where the cases are being weird?

Reply:
No.. aria-hidden is correctly on the tooltip.. maybe a shadowdom issue?

Reply:
yeah, it seems like like a shadow dom thing whatever it is